Sources/NIO/Bootstrap.swift

```diff
@@ -163,6 +163,41 @@ public final class ServerBootstrap {
         }
     }
 
+    /// Use the existing bound socket file descriptor.
+    ///
+    /// - parameters:
+    ///     - descriptor: The _Unix file descriptor_ representing the bound stream socket.
+    public func withBoundSocket(descriptor: Int32) -> EventLoopFuture<Channel> {
```

Comment:
nit: Should we use `CInt` here ?

Reply:
The existing function:

```
    static func newSocket(protocolFamily: Int32, type: CInt, setNonBlocking: Bool = false) throws -> Int32 {
```

I was trying to be compatible with that. If we're `CInt`ing withBoundSocket, we probably need to `CInt` `newSocket` as well, don't you think?